### PR TITLE
Adding G0W0 Approximation feature

### PR DESCRIPTION
**Pull request by sblisesivdin**
Only for gpawsolve.py. Not working with gg.py

### CODE REVIEW

File: gpawsolve.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 gpawsolve.py: High-level Calculation Script for GPAW
 More information: $ python gpawsolve.p -h
  '''
+import pickle
```
Comment (beyzalisesivdin) [inline gpawsolve.py]: No need to use pickle

**Review by beyzalisesivdin — COMMENTED** [review]
I made the change.

**Comment by sblisesivdin** [comment]
I am merging as is @beyzalisesivdin . Please make that change again.

**Comment by beyzalisesivdin** [comment]
They are already done. No need to change it again.

**Pull request merged**